spec/spec_helper_acceptance.rb

```diff
@@ -1,16 +1,14 @@
-require 'beaker-rspec/spec_helper'
-require 'beaker-rspec/helpers/serverspec'
-require 'beaker/puppet_install_helper'
-require 'beaker/module_install_helper'
+require 'open3'
+require 'json'
+require 'yaml'
 
-device_hostname = 'target'
-
-cisco_host = find_at_most_one('cisco_host')
+cisco_host = YAML.safe_load(File.read('hosts.yaml'))
```

Comment:
One last thing, for local execution the following might make it more robust, as currently it will fail unless you have a `hosts.yaml` file:

```
cisco_host = YAML.safe_load(File.read(ENV['BEAKER_setfile'] || 'hosts.yaml')) if File.exists?(ENV['BEAKER_setfile'] || 'hosts.yaml')
```